.travis.yml

```diff
@@ -5,12 +5,17 @@ jobs:
   include:
     - stage: analyze
       name: "SDK: dev; PKG: example; TASKS: `dartanalyzer .`"
-      dart: dev
+      dart: stable
       env: PKGS="example"
       script: ./tool/travis.sh dartanalyzer
+    - test:
+      name: "test generator"
+      dart: stable
+      script: "cd generator && pub get && pub run test"
 
 stages:
   - analyze
+  - test
 
```

Comment:
Add a new stage for `formatting` this will check the formatting of all dart files using `dartfmt . -n --set-exit-if-changed --fix`. This will also help to increase the score on pub.